test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,33 @@
+package pages;
+
+import abstractPackage.BasePageClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class LoginPage extends BasePageClass {
+
+
+    private String url = "https://findlaw1.flsitebuilder-qa.com/wp-admin/";
+    public final String email = "nastassia.chaliapina@thomsonreuters.com";
+    public final String password = "7845366Nastya";
```

Comment:
1. password really shouldn't be stored in class. use properties, db etc
2. if you still leave it here, it should be private in test class, as well as email. You store them on the page, but call them from test? - bad from architectural point of view and from security.